hotsos/plugin_extensions/openstack/summary.py

```diff
@@ -7,8 +15,24 @@ class OpenstackSummary(OpenstackServiceChecksBase):
 
     @idx(0)
     def __summary_release(self):
+        self.check_eol()
+
         return self.release_name
 
+    def check_eol(self):
+        if self.release_name != 'unknown':
+            eol = OST_EOL_INFO[self.release_name]
+            today = datetime.now()
+            delta = (eol - today).days
+
+            if delta <= 0:
+                IssuesManager().add(
+                    OpenstackError("{} is EOL".format(self.release_name)))
```

Comment:
Once this is moved to a scenario I think we should currently only raise an issue for releases that are already EOL (not ones that are about to be). This is is just to reduce noise. Using the summary info we would always be able to tell exactly when EOL will be if required.